robosuite/models/assets/base.xml

```diff
@@ -1,7 +1,7 @@
 <!-- This is the base xml for all physics simulations. Set global configs here. -->
 <mujoco model="base">
   <compiler angle="radian" meshdir="meshes/" inertiagrouprange="0 0"/>
-  <option impratio="20" cone="elliptic"/>
+  <option impratio="20" cone="elliptic" density="1.2" viscosity="0.00002"/>
```

Comment:
do you know if this will change the default dynamics from v1.1, and cause action playback to fail for demos collected on that version?